fieldservice/models/fsm_order.py

```diff
@@ -114,6 +114,9 @@ def _track_subtype(self, init_values):
     location_id = fields.Many2one(
         "fsm.location", string="Location", index=True, required=True
     )
+    location_id_owner_id = fields.Char(
+        related="location_id.owner_id.name", string="Location Related Owner"
+    )
```

Comment:
```suggestion
    location_owner_id = fields.Many2one(
        related="location_id.owner_id", string="Location Related Owner"
    )
```

There's no point in using the `name`, use the related many2one directly

Reply:
Done! Thanks for your feedback. Can you review again please